lib/driver.js

```diff
@@ -128,7 +129,7 @@ class AndroidDriver extends BaseDriver {
     if (this.opts.ignoreUnimportantViews) {
       await this.settings.update({ignoreUnimportantViews: this.opts.ignoreUnimportantViews});
     }
-    if (!this.appOnDevice) {
+    if (!this.appOnDevice && this.opts.autoLaunch) {
```

Comment:
Can we add a comment here about autoLaunch, that user is responsible for init and startapp?